synapse/federation/transport/server.py

```diff
@@ -649,7 +649,9 @@ async def on_GET(
     ) -> Tuple[int, JsonDict]:
         try:
             # Retrieve the room versions the remote homeserver claims to support
-            supported_versions = parse_strings_from_args(query, "ver", encoding="utf-8")
+            supported_versions = parse_strings_from_args(
+                query, "ver", required=True, encoding="utf-8"
+            )
```

Comment:
This is meant to be a required parameter, and if its not supplied then the code will explode later anyway.

Reply:
Good shout. We can actually remove the try-except in this case too!